crates/sui-protocol-config/src/lib.rs

```diff
@@ -263,6 +263,7 @@ const MAX_PROTOCOL_VERSION: u64 = 96;
 // Version 96: Enable authority capabilities v2.
 //             Fix bug where MFP transaction shared inputs' debts were not loaded
 //             Create Coin Registry object
+// Version 97: Add constant for event emit auth stream cost.
```

Comment:
Nit: You also added authentication stream support to the sui-framework and sui-framework upgrades are (implicitly) a part of the protocol versions so we try to include that here as well (e.g., just a simple "Added support to sui-framework for authenticated event streams" is fine).